src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -22,6 +22,9 @@
  */
 class DateTimeToLocalizedStringTransformer extends BaseDateTimeTransformer
 {
+    // Midnight of last day of 9999 to prevent 5 digit years from being transformed
+    const MAX_TIMESTAMP = 253402214400;
```

Comment:
Should be a `private static` variable so that we won't have to maintain a new public const.

Reply:
or just placed inlined as it is used only once (a `private static` would force the compiler to consider it mutable)

Reply:
Agreed, sucks that we don't have `private const` back in 2.7.